README.rst

```diff
@@ -88,6 +93,7 @@ Freenode)`_
 .. _Let’s Encrypt: https://letsencrypt.org
 .. _UNIX philosophy: https://en.wikipedia.org/wiki/Unix_philosophy
 .. _IRC (#simp\_le at Freenode): http://webchat.freenode.net?randomnick=1&channels=%23simp_le&prompt=1
+.. _simp\_le for Docker: docker
```

Comment:
This doesn't render correctly; it tries to link to a blob (regular file) called docker, but it's a directory. You probably want the README.

Reply:
I intended to link to the directory (to see both the readme and the files when you clink on the link), it appeared to work when I tried it, is this actually something to avoid ?

On a sidenote it seems escaping the underscores isn't needed on GitHub.